README.md

```diff
@@ -158,6 +153,10 @@ In order for a yaml file to be considered "valid", the following conditions must
     * Every character with `mission_importance` should be an entry in `character_importance`, and vice-versa 
     * This does not include "normal", which is the default level of importance. For example, a character may not specify `mission_importance` and `character_importance` may explicitly specify the character with importance "normal", or a character may specify `mission_importance` with "normal" and `character_importance` may not list that character 
 
+#### Eval Mode
+When running in eval mode (-e), additional checks are implemented:
+* No supplies or treatments are allowed that are not in the simulator (no blankets, epi pens, or vented chest seals)
+
```

Comment:
If we're going eval as the default and with -t / --train, then just update this to say "(the default)" or "When not running in training mode (-t), ..."